cli/commands/plugins_update.go

```diff
@@ -0,0 +1,30 @@
+package commands
+
+import (
+	"github.com/hasura/graphql-engine/cli"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func newPluginsUpdateCmd(ec *cli.ExecutionContext) *cobra.Command {
+	pluginsUpdateCmd := &cobra.Command{
+		Use:          "update",
```

Comment:
This command need not be a separate one, update plugin index as part of `plugins list` -- show a warning if updating fails, give a flag to skip updating index.